tests/conftest.py

```diff
@@ -56,3 +56,39 @@ def disable():
 @pytest.fixture(scope="module")
 def reporter():
     return MinimalTestReporter
+
+
+def pytest_addoption(parser) -> None:
+    parser.addoption(
+        "--primer_stdlib",
```

Comment:
```suggestion
        "--primer-stdlib",
```

Reply:
You can't mark with `@pytest.mark.primer_stdlib` so I thought I would keep consistency throughout the functions using it. Do you want me to change it anyway?

Reply:
For CLI option I think the standard is ``-``